lib/middleware/schemas/templates.schema.json

```diff
@@ -7,6 +7,11 @@
   "default": {},
   "examples": [],
   "properties": {
+    "securityAttribute":{
```

Comment:
It would be great to add an enum here, listing the allowed values.

ie:
```"enum": [
            "textField",
            "textArea",
            "dropdown",
            "radio",
            "checkbox",
            "fileInput",
            "dynamicRow",
            "richText"
          ]
          ```

Reply:
Absolutely ! it will improve the readability as well.